### PR TITLE
Fixed #25267 -- Corrected message for admin.E122 error.

### PR DESCRIPTION
**Pull request by zauddelig**
https://code.djangoproject.com/ticket/25267

### CODE REVIEW

**Comment by timgraham** [comment]
Could you please add a test in `tests/admin_checks`? You might use the error message as documented in `docs/ref/checks.html`. See our [patch review checklist](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#patch-review-checklist) for the commit message format. Thanks.

**Comment by zauddelig** [comment]
I added the tests sorry I was unsure on how document the commit and I wasn't sure were the check tests were.

File: django/contrib/admin/checks.py [hunk 1]
```diff
@@ -774,8 +774,16 @@ def _check_list_editable_item(self, cls, model, field_name, label):
                                           model=model, obj=cls, id='admin.E121')
         else:
             if field_name not in cls.list_display:
-                return refer_to_missing_field(field=field_name, option=label,
-                                              model=model, obj=cls, id='admin.E122')
+                return [
+                    checks.Error(
+                        "The value of '%s' refers to %s but it is not defined in 'list_display'." % (
```
Comment (timgraham) [inline django/contrib/admin/checks.py]: documentation in docs/ref/checks.txt says "which is not contained in 'list_display'".

**Comment by timgraham** [comment]
Please squash commits when updating, thanks.

**Comment by timgraham** [comment]
merged in ece78684d9e09c477c4a0f10236c0aec45bac726, thanks!

**Pull request closed without merging**